src/archanaNJun20/IndexOfExample2.java

```diff
@@ -0,0 +1,26 @@
+package archanaNJun20;
+
+public class IndexOfExample2 {
+
+	void printIndexOf(String input)
+	{
+		for(int index=0;index<input.length();index++){
+			char ch=input.charAt(index);
+			int count=0;
+			if(index==input.indexOf(ch))
+				for(int index1=0;index1<input.length();index1++) {
+					if(ch==input.charAt(index1)){
+						count++;
+					}
+				}
+			if(count>1){
```

Comment:
You have increased the complexity in code, Using one for loop and one if block you can achieve the output.   